packages/commonwealth/client/scripts/views/components/sidebar/CommunitySection/CommunitySection.tsx

```diff
@@ -73,7 +75,7 @@ export const CommunitySection = ({ showSkeleton }: CommunitySectionProps) => {
                 voteWeight={currentVoteWeight}
                 stakeNumber={stakeBalance}
                 stakeValue={stakeValue}
-                denomination="ETH"
+                denomination={findDenominationString(activeChainId) || 'ETH'}
```

Comment:
i don't know if returning `ETH` by default is required, if yes, then lets update `findDenominationString` to return it by default, and update all instances.